app/src/pages/playground/PlaygroundDatasetSection.tsx

```diff
@@ -82,32 +105,48 @@ export function PlaygroundDatasetSection({
               </Text>
             ) : null}
           </Flex>
-          {experimentIds.length > 0 && (
-            <LinkButton
-              size="S"
-              isDisabled={isRunning}
-              leadingVisual={
-                <Icon
-                  svg={
-                    isRunning ? (
-                      <Icons.LoadingOutline />
-                    ) : (
-                      <Icons.ExperimentOutline />
-                    )
+          <Flex direction="row" gap="size-100" alignItems="center">
+            <EvaluatorSelect
```

Comment:
I would kind of expect to see selected evaluators in a row next to this button. Even if it is just the evaluator name for now. We can follow up on this in a new PR though.